qiskit_nature/operators/second_quantization/fermionic_op.py

```diff
@@ -282,94 +385,113 @@ def add(self, other: "FermionicOp") -> "FermionicOp":
         return FermionicOp(
             self._data + other._data,
             max(self.register_length, other.register_length),
-            self.sparse_label or other.sparse_label,
+            self.display_format or other.display_format,
         )
 
-    def to_list(self) -> List[Tuple[str, complex]]:
+    # pylint: disable=arguments-differ
+    def to_list(
+        self,
+        display_format: Optional[str] = None,
+    ) -> List[Tuple[str, complex]]:  # type: ignore
         """Returns the operators internal contents in list-format.
 
+        Args:
+            display_format: If sparse, the label is represented sparsely during output.
+                            if dense, the label is represented densely during output. (default: None)
```

Comment:
```suggestion
            display_format: when specified this will overwrite ``self.display_format``. Can
                be either 'dense' or 'sparse'. See the class documentation for more details.
```